README.md

```diff
@@ -89,6 +93,17 @@ viewed by clicking `View app`.
 
     ![Delivery Pipeline](doc/source/images/ibm-cloud-deploy.png)
 
+## Deploy on Kubernetes
+
+You can also deploy the model and web app on Kubernetes using the latest docker images on Docker Hub.
+
+On your Kubernetes cluster, run the following commands:
+
+    kubectl apply -f https://raw.githubusercontent.com/IBM/MAX-Image-Caption-Generator/master/image-caption-generator.yaml
+    kubectl apply -f https://raw.githubusercontent.com/IBM/MAX-Image-Caption-Generator-Web-App/master/image-caption-generator-web-app.yaml
+
+The model will be accessible at port `5000` and the web app at port `8088` of your cluster.
```

Comment:
Strictly speaking, the model is not publicly accessible at port `5000` (or at least it appears so in my Kube dashboard that it does not have an external endpoint).

![screen shot 2018-07-04 at 13 30 13](https://user-images.githubusercontent.com/1036807/42274851-7c1f9b9e-7f8e-11e8-8ceb-20cde3472eb4.png)

Reply:
I tried to word it to make it clear that they're the internal ports, but I can reword it if that's not clear.